website/src/categories/components/dialog/code.md

```diff
@@ -7,29 +7,68 @@ eleventyNavigation:
   key: DialogCode
 ---
 <section>
```

Comment:
Can you please add installation section to the dialog like in the switch PR https://github.com/sl-design-system/components/pull/953